service/http.directory/src/test/scripts/org/kaazing/gateway/service/http/directory/get.nonexistent.page.check.http.keepalive.timeout.rpt

```diff
@@ -20,20 +20,12 @@
 #
 
 # get.nonexistent.page.check.status.code.404
-connect tcp://localhost:8003
+connect http://localhost:8003/nonexistentpage.html
 connected
 
-write "GET /nonexistentpage.html HTTP/1.1\r\n"
-write "Host: localhost:8003\r\nUser-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.6; rv:8.0) Gecko/20100101 Firefox/8.0\r\n"
-write "Accept: text/html, application/xhtml+xml, application/xml;q=0.9,*/*;q=0.8\r\n\r\n"
-write notify BARRIER
+write method "GET"
 
 #We need to wait for all the data to be written first
-read await BARRIER
-read "HTTP/1.1 404 Not Found\r\n"
-read "Content-Type: text/html\r\n"
-read "Content-Length: 61\r\n"
-read "\r\n"
-read "<html><head></head><body><h1>404 Not Found</h1></body></html>"
+read status "404" "Not Fou"
+
 #wait for server to close connection due to http.keepalive.timeout
```

Comment:
Need to keep the closed to ensure the server is actually closing the connection.  This means you will need to read the response as well.

```
read "HTTP/1.1 404 Not Found\r\n"
```